velox/docs/configs.rst

```diff
@@ -633,10 +633,6 @@ Each query can override the config by setting corresponding query session proper
      - string
      -
      - The GCS storage endpoint server.
```

Comment:
Maybe should mention that this is a URI that includes the scheme rather than just the server? This PR changes the behavior of this variable. Before it did require a scheme-less hostname (+port).

Reply:
Good point. I added the endpoint default as `https://storage.googleapis.com` since it is the most common.
I also updated the doc.